cloud-function/lambda/js.md

```diff
@@ -23,7 +23,7 @@ A lambda function can be created using the `@skygear.op` decorator.
 The method is:
 
 ```javascript
-op(name: String, func: function(param: Object, options: *), authRequired: Boolean, userRequired: Boolean)
+op(name: string, func: (param: object, options: any), options: { userRequired?: boolean, keyRequired?: boolean})
```

Comment:
@tensiuyan The original copy seems only the options part is not correct?

https://docs.skygear.io/js/reference/latest/function/index.html#static-function-op